docs/environments/icp.md

```diff
@@ -0,0 +1,49 @@
+# IBM Cloud Private Deployment Guide
+
+Federation v2 can be deployed to and manage [IBM Cloud Private](https://www.ibm.com/cloud/private) clusters.
+As Federation v2 requests Kubernetes v1.11 or greater, so please make sure deploy IBM Cloud Private 3.1.1
+or higher version.
+
+The following example deploys two IBM Cloud Private 3.1.1 clusters named as `cluster1` and `cluster2`.
+
+## Install IBM Cloud Private
+
+Please follow the [Guidance in IBM Cloud Private 3.1.1 Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install.html)
+to install.
+
+**NOTE:** As we need to install two clusters named as `cluster1` and `cluster2`, so after `cluster/config.yaml`
+generated, update the cluster name to `cluster1` and `cluster2` for the two clusters before install.
```

Comment:
`s/generated/is generated/`
`s/update the cluster name.*/update the names of the 2 clusters to 'cluster1' and 'cluster2' before installing federation`